test/app/actions/async.test.js

```diff
@@ -121,9 +113,16 @@ describe('Asynchronous Actions', () => {
           meta: {source: actionSources[actionTypes.SET_NEW_PATIENT_URL]}
         },
         {
-          type: actionTypes.SET_PAGE,
-          payload: {page: pages.LOGIN},
-          meta: {source: actionSources[actionTypes.SET_PAGE]}
+          type: '@@router/CALL_HISTORY_METHOD',
```

Comment:
Almost all the `SET_PAGE` actions are now `@@router/CALL_HISTORY_METHOD`, which represents a page navigation.